WebDriverAgentLib/FBApplication.m

```diff
@@ -35,21 +35,44 @@ @interface FBApplication ()
 @implementation FBApplication
 
 + (instancetype)fb_activeApplication
+{
+  return [self fb_activeApplicationWithDefaultBundleId:nil];
+}
+
++ (instancetype)fb_activeApplicationWithDefaultBundleId:(nullable NSString *)bundleId
 {
   NSArray<XCAccessibilityElement *> *activeApplicationElements = [FBXCAXClientProxy.sharedClient activeApplications];
-  XCAccessibilityElement *activeApplicationElement = [activeApplicationElements firstObject];
+  XCAccessibilityElement *activeApplicationElement = nil;
   if (activeApplicationElements.count > 1) {
```

Comment:
Shouldn't this be now bigger than 0 instead of 1?

Reply:
it is fine. If it equals to 1 then the first element element is set in `else` block. If the length is zero then firstObject will return nil, which will lead to an exception